[IMP] website/ecommerce: update catalog page (#15569)
odoo/documentation: content/applications/sales/sales/products_prices/products/variants.rst

   other similar records in Odoo.

.. seealso::

>    :ref:`Product variants in the eCommerce shop <ecommerce/categories_variants/product-variants>`

Reviewer: eCommerce shop is a bit redundant I think so I'd change this to eCommerce or online shop.

Type` determines how this product is shown on the online store, *Point of Sale* dashboard, and
*Product Configurator*.

The following :guilabel:`Display Type` options are available.

.. tabs::

   .. tab:: :guilabel:`Pills`

      Options appear as selectable buttons.

      .. image:: variants/pills-display-type.png
         :alt: Pills display type

   .. tab:: :guilabel:`Color`

      Options appear as small, colored circles, which reflect any HTML color codes.

      .. image:: variants/color-display-type.png
         :alt: Pills display type

   .. tab:: :guilabel:`Radio`

      Options appear in a bullet-style list.

      .. image:: variants/radio-display-type.png
         :alt: Pills display type

   .. tab:: :guilabel:`Select`

      Options appear in a drop-down menu.

      .. image:: variants/select-display-type.png
         :alt: Pills display type

   .. tab:: :guilabel:`Image`

      Options appear as images.

      .. image:: variants/image-display-type.png
         :alt: Pills display type

   .. tab:: :guilabel:`Multi-checkbox`

      Options appear as selectable checkboxes. This allows customers to choose options for

>       themselves and is good choice for highly customizable products

Reviewer: ```suggestion
      Options appear as selectable checkboxes. This allows customers to choose multiple options and is ideal for highly customizable products, e.g. to add extras to food orders.
```
(if you find another example for ecommerce or a better example period, feel free to replace the last part of the sentence!)

Author: Maybe to add extra prints on customizable shirts? Or extra features like additional drawers when configuring a closet...

      .. image:: variants/multi-checkbox-display-type.png
         :alt: Pills display type

The :guilabel:`Variant Creation` field informs Odoo when to automatically create a new variant

> once an attribute is added to a product.

Reviewer: Sorry, I know you didn't write this but I think we can really improve this sentence as follows:

```suggestion
The :guilabel:`Variant Creation` field determines if/when a new variant is automatically created
once an attribute is added to a product.
```

.. note::
   The :guilabel:`Variant Creation` field **must** be set to :guilabel:`Never` in
   order for the :guilabel:`Multi-checkbox` to work properly as the :guilabel:`Display

>    Type`.

Reviewer: I think I would make this an important note and rephrase slightly:

```suggestion
.. important::
   If the attribute's :guilabel:`Display Type` is :guilabel:`Multi-checkbox`, you **must** set the :guilabel:`Variant Creation` field to :guilabel:`Never`.
```
